### PR TITLE
ndk-multilib: Exclude native stubs for shared libs

### PR DESCRIPTION
**Pull request by xtkoba**
which are now installed with subpackage named ndk-multilib-native-stubs.

### CODE REVIEW

**Comment by buttaface** [comment]
This moves the libraries for all arches, breaking on-device cross-compilation to x86_64 from aarch64. That's why I think a post-install hook is best, provided it can detect and only move the four native libraries.

File: packages/ndk-multilib/postinst-header.in [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!@TERMUX_PREFIX@/bin/sh
+
+pkg_format="@TERMUX_PACKAGE_FORMAT@"
+pkg_arch=unknown
+case "$pkg_format" in
+	debian ) pkg_arch=$(dpkg --print-architecture) ;;
+	pacman ) pkg_arch=$(sed -En 's/^\s*Architecture\s*=\s*(aarch64|arm|i686|x86_64)\s*$/\1/p' @TERMUX_PREFIX@/etc/pacman.conf) ;;
```
Comment (xtkoba) [inline packages/ndk-multilib/postinst-header.in:7]: I'm not familiar with pacman and I cannot figure out a more appropriate way to determine the native architecture. @Maxython What do you think?

Comment (Maxython) [reply packages/ndk-multilib/postinst-header.in:7]: You can do that:
```bash
pkg_arch=$(pacman-conf Architecture)
```

Comment (xtkoba) [reply packages/ndk-multilib/postinst-header.in:7]: Thanks!

**Comment by buttaface** [comment]
Tested this out, worked well, thanks for working on this.

One small tweak: I notice the stubs package has no files, can you add a notice in there, just saying what it is or something? The reason is that I'm planning an upcoming pull to allow building and installing arch-specific libraries on all platforms, so you can cross-compile with other libraries than just ndk-multilib.

However, this will depend on removing subpackages with no files, because armv7 libraries won't be built on the aarch64 CI run, but you are creating such an empty package now. If you just add a single file, that should present no problem for my upcoming pull and I assume we don't have any other such empty packages already.

**Pull request merged**